docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,451 @@
+
+* Deploy the resources using the provided YAML and `kubectl apply -f vpc-workflow.yaml`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: VPC
+metadata:
+  name: ack-vpc
+spec:
+  cidrBlocks: 
+  - 10.0.0.0/16
+  enableDNSSupport: true
+  enableDNSHostnames: true
+  tags:
+    - key: purpose
+      value: ack-tutorial
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: InternetGateway
+metadata:
+  name: ack-igw
+spec:
+  vpcRef:
+    from:
+      name: ack-vpc
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: NATGateway
+metadata:
+  name: ack-natgateway1
+spec:
+  subnetRef:
+    from:
+      name: ack-public-subnet1
+  allocationRef:
+    from:
+      name: ack-eip1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: ElasticIPAddress
+metadata:
+  name: ack-eip1
+spec:
+  tags:
+    - key: purpose
+      value: ack-tutorial
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: RouteTable
+metadata:
+  name: ack-public-route-table
+spec:
+  vpcRef:
+    from:
+      name: ack-vpc
+  routes:
+  - destinationCIDRBlock: 0.0.0.0/0
+    gatewayRef:
+      from:
+        name: ack-igw
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: RouteTable
+metadata:
+  name: ack-private-route-table-az1
+spec:
+  vpcRef:
+    from:
+      name: ack-vpc
+  routes:
+  - destinationCIDRBlock: 0.0.0.0/0
+    natGatewayRef:
+      from:
+        name: ack-natgateway1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: ack-public-subnet1
+spec:
+  availabilityZone: us-west-2a
+  cidrBlock: 10.0.0.0/20
+  mapPublicIPOnLaunch: true
+  vpcRef:
+    from:
+      name: ack-vpc
+  routeTableRefs:
+  - from:
+      name: ack-public-route-table
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: ack-private-subnet1
+spec:
+  availabilityZone: us-west-2a
+  cidrBlock: 10.0.128.0/20
+  vpcRef:
+    from:
+      name: ack-vpc
+  routeTableRefs:
+  - from:
+      name: ack-private-route-table-az1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: SecurityGroup
+metadata:
+  name: ack-security-group
+spec:
+  description: "ack security group"
+  name: ack-sg
+  vpcRef:
+     from:
+       name: ack-vpc
+  ingressRules:
+    - ipProtocol: tcp
+      fromPort: 22
+      toPort: 22
+      ipRanges:
+        - cidrIP: "0.0.0.0/0"
+          description: "ingress"
+```
+ 
+ 
+{{% hint type="info" title="Referencing Resources" %}}
+Notice that the ACK custom resources reference each other using "*Ref" fields inside the manifest and the user does not have to worry about finding `vpc-ID` when creating the Subnet resource manifests.
```

Comment:
I might call this out before showing the manifest, so that people are able to read through the manifest with an understanding of resource references in mind

Reply:
that makes sense! will move it before the manifest.